src/helpers/verifyToken.js

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken';
+
+const verifyToken = (token) => jwt.verify(token, process.env.JWT_KEY, (err, decoded) => {
+  if (err) {
+    return false;
+  }
+  return decoded;
+});
+
+export default verifyToken;
```

Comment:
This might not work for me as we all need this file, because I am expecting the data when the token is valid, I recommend using `decode` instead.

Reply:
How sure are you @william000000 that token matches with our secret key?

Reply:
I think you are right @NiyoEric, please go ahead with this!